spec/model/reservation_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
```

Comment:
- It looks like there are some issues to fix on tests in general, I'll add here the failed test, please, I invite you to fix all of this. 🐧 

```
30 examples, 12 failures, 1 error occurred outside of examples

Failed examples:

rspec ./spec/requests/api/v1/cars_spec.rb:19 # api/v1/cars /api/v1/cars post successful returns a 200 response
rspec ./spec/requests/api/v1/cars_spec.rb:34 # api/v1/cars /api/v1/cars/new get successful returns a 200 response
rspec ./spec/requests/api/v1/cars_spec.rb:52 # api/v1/cars /api/v1/cars/{id}/edit get successful returns a 200 response
rspec ./spec/requests/api/v1/cars_spec.rb:73 # api/v1/cars /api/v1/cars/{id} get name found returns a 200 response
rspec ./spec/requests/api/v1/cars_spec.rb:88 # api/v1/cars /api/v1/cars/{id} get Car not found returns a 404 response
rspec ./spec/requests/api/v1/reservations_spec.rb:19 # api/v1/reservations /api/v1/cars/{car_id}/reservations get successful returns a 200 response
rspec ./spec/requests/api/v1/reservations_spec.rb:42 # api/v1/reservations /api/v1/cars/{car_id}/reservations post successful returns a 200 response
rspec ./spec/requests/api/v1/reservations_spec.rb:62 # api/v1/reservations /api/v1/cars/{car_id}/reservations/new get successful returns a 200 response
rspec ./spec/requests/api/v1/reservations_spec.rb:83 # api/v1/reservations /api/v1/cars/{car_id}/reservations/{id}/edit get successful returns a 200 response
rspec ./spec/requests/api/v1/reservations_spec.rb:121 # api/v1/reservations /api/v1/cars/{car_id}/reservations/{id} patch successful returns a 200 response
rspec ./spec/requests/api/v1/reservations_spec.rb:137 # api/v1/reservations /api/v1/cars/{car_id}/reservations/{id} put successful returns a 200 response
rspec ./spec/requests/api/v1/reservations_spec.rb:153 # api/v1/reservations /api/v1/cars/{car_id}/reservations/{id} delete successful returns a 200 response
```